icepyx/tests/test_visualization.py

```diff
@@ -0,0 +1,16 @@
+from icepyx.core.visualization import Visualize
```

Comment:
Need to import pytest to fix the broken Travis CI test

```suggestion
import pytest

from icepyx.core.visualization import Visualize
```